tests/scripts/tesseract.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
-mkdir ~/tesseract
-cd ~/tesseract
-wget http://alpha.library.yorku.ca/tesseract.tar.gz
-wget http://alpha.library.yorku.ca/tessdata.tar.gz
-tar xf tesseract.tar.gz && rm -rf tesseract.tar.gz
-dpkg -i tesseract/libleptonica.deb
-dpkg -i tesseract/tesseract-ocr.deb
-tar xf tessdata.tar.gz && rm -rf tessdata.tar.gz
-cp tessdata/* /usr/local/share/tessdata/
-sudo ldconfig
-echo -e "\ntesseract output:"
+if [[ $TRAVIS_PHP_VERSION == '5.3.3' ]]; then
+  mkdir ~/tesseract
+  cd ~/tesseract
+  wget http://alpha.library.yorku.ca/tesseract.tar.gz
+  wget http://alpha.library.yorku.ca/tessdata.tar.gz
+  tar xf tesseract.tar.gz && rm -rf tesseract.tar.gz
+  dpkg -i tesseract/libleptonica.deb
+  dpkg -i tesseract/tesseract-ocr.deb
+  tar xf tessdata.tar.gz && rm -rf tessdata.tar.gz
+  cp tessdata/* /usr/local/share/tessdata/
+  sudo ldconfig
+  echo -e "\ntesseract output:"
+else
+  sudo apt-get -y install tesseract-ocr tesseract-ocr-eng tesseract-ocr-fra
+fi
 
-tesseract --version && tesseract --list-langs && cd ~ && rm -rf ~/tesseract
+tesseract --version && tesseract --list-langs
```

Comment:
Should you move the `cd ~ && rm -rf ~/tesseract` up into the end of the `if [[ $TRAVIS_PHP_VERSION == '5.3.3' ]]; then` block?

Reply:
Actually not sure why we need to `cd ~` and then `rm ~/tesseract` seems like just the `rm` would work.

Reply:
I think we don't even need that. Travis images get wiped out after they finish so i removed it completely because it seemed to me like a waste of code? or time? or cpu? or energy?